digestparser/json_output.py

```diff
@@ -0,0 +1,183 @@
+"build JSON output from digest content"
+import os
+import time
+import copy
+from collections import OrderedDict
+import requests
+from elifetools.utils import copy_attribute
+from digestparser.utils import msid_from_doi
+from digestparser.jats import (parse_jats_file, parse_jats_digest, parse_jats_pub_date,
+                               parse_jats_subjects, xml_to_html)
+from digestparser.build import build_digest
+
+
+def iiif_server_info(info_url):
+    "get the image info from the IIIF server"
+    info = {}
+    try:
+        response = requests.get(info_url)
+        info = response.json()
+    except:
+        # could be any error right now
```

Comment:
Ok for exploring what may happen, but log the error with a `warn` level; we can get more specific later and catching only `requests`-related exceptions

Reply:
There wasn't any logging in this library yet, but I found logging in another library I sort of followed that. Logging the exception as an exception (instead of warn) if that makes sense.

I hope it satisfies the comment for now, if not please send some additional feedback.

Reply:
What I mean is we are hiding `response` here so if the empty `info` causes errors down the line we don't have a way to check what was e.g. the original status code?